File: src/java/test/src/androidTest/java/com/tencent/wcdbtest/base/WinqTool.java
package com.tencent.wcdbtest.base;

import com.tencent.wcdb.winq.Identifier;

import org.junit.Assert;

public class WinqTool {
     public static void winqEqual(Identifier winq, String sql) {
          Assert.assertEquals(winq.getDescription(), sql);
     }
}

File: src/rust/wcdb_rust/tests/base/winq_tool.rs
use wcdb_core::winq::identifier::IdentifierTrait;

pub struct WinqTool {}

impl WinqTool {
    pub fn winq_equal(winq: &impl IdentifierTrait, sql: &str) {
        assert_eq!(sql, winq.get_description())
    }
}
